### DISCUSSION TITLE
It is asking to input value for instance

### DISCUSSION THREAD

**Discussion by Nazrina412** [learndevopsonline/learndevopsonline#1944] [Q&A]
![image](https://github.com/user-attachments/assets/824a4019-cdde-4ec4-bc38-f75bb120a1b0)

**Comment by chaithanya1812** [comment]
## confusing
![image](https://github.com/user-attachments/assets/241f0ccb-18ba-4c36-887c-4aa374ebc697)

**Comment by Nazrina412** [comment]
Hi.,

I added the files still it is asking for the value
[image: image.png]

**Reply by chaithanya1812** [reply]
## change the file content.
![image](https://github.com/user-attachments/assets/02fc08a0-f871-4c98-9205-fb117e9597ec)

**Comment by Nazrina412** [comment]
I am still getting same error I created module in readme
[image: image.png]

**Comment by Nazrina412** [comment]
Hi I am trying but still I am not getting. I have declared variables both
[image: image.png]

**Comment by Nazrina412** [comment]
I am getting this error after trying several times
[image: image.png]

**Comment by Nazrina412** [comment]
[image: image.png]
